tfx/orchestration/kubeflow/kubeflow_dag_runner.py

```diff
@@ -347,6 +350,13 @@ def _construct_pipeline_graph(self, pipeline: tfx_pipeline.Pipeline,
 
       component_to_kfp_op[component] = kfp_component.container_op
 
+    # If exit handler defined create an exit handler and add all ops to it.
+    if self._exit_handler:
+      exit_op = component_to_kfp_op[self._exit_handler]
+      with dsl.ExitHandler(exit_op) as exit_handler_group:
+        exit_handler_group.name = 'tfx-exit-handler'
```

Comment:
for consistency with V2, we may call it '_tfx_dag'
(note:variable for v2 is defined under orchestration/kubeflow/v2/compiler_utils.py)